specs/engine.spec/src/main/resources/META-INF/zilla/core.idl

```diff
@@ -100,4 +100,36 @@ scope core
             octets extension;
         }
     }
+
+    scope event
+    {
+        enum Level
+        {
+            INFO,
+            WARNING,
+            ERROR
+        }
+
+        enum Result
+        {
+            SUCCESS,
+            FAILURE
+        }
+
+        struct Event
+        {
+            Result result;
+            Level level;
+            int64 timestamp = 0;
+        }
+
+        struct BindingEvent extends Event
+        {
+            int64 originId;
+            int64 routedId;
+            int64 initialId;
+            int64 replyId;
+            int64 traceId;
```

Comment:
I suspect that `traceId` will be needed in the generic event and that may actually be sufficient even for binding events initially.

Please take a look at what we'll need for `otlp` exporter of events to provide sufficient context.